semanticdb-java/src/main/protobuf/semanticdb.proto

```diff
@@ -378,6 +379,22 @@ enum BinaryOperator {
   GREATER_THAN_EQUAL = 17;
   LESS_THAN_EQUAL = 18;
 }
+
+message UnaryOperatorTree {
+  UnaryOperator op = 1;
+  Tree tree = 2;
+}
+
+enum UnaryOperator {
```

Comment:
Might be worth linking to JLS to document that this is a complete representation for Java 8 https://docs.oracle.com/javase/specs/jls/se8/html/jls-15.html#jls-15.15